tasks/33_array-plus-array/array-plus-array.js

```diff
@@ -0,0 +1,10 @@
+export function arrayPlusArray(arr1, arr2) {
+  const arr3 = [];
+  const maxLength = Math.max(arr1.length, arr2.length);
+  for (let i = 0; i < Math.min(arr1.length, arr2.length); i++) {
+    arr3.push(arr1[i] + arr2[i]);
+  }
+  arr1 = arr1.slice(maxLength);
+  arr2 = arr2.slice(maxLength);
+  return [...arr3, ...arr1, ...arr2];
```

Comment:
Let's make it even more simple. How about this:
```
  const arr3 = [];
  const maxLength = Math.max(arr1.length, arr2.length);;

  for (let i = 0; i < maxLength; i++) {
    arr3.push((arr1[i] || 0) + (arr2[i] || 0));
  }

  return arr3;
```
Is it readable?

Also, is there additional way to detect `maxLength` using one line too?

Reply:
I used your code because it is simpler and more readable. And for determining the `maxLength` i used a one-line `ternary` operator.